app/liquid_tags/podcast_tag.rb

```diff
@@ -26,19 +26,14 @@ def initialize(_tag_name, link, _parse_context)
     super
     @episode = fetch_podcast(link)
     @podcast ||= Podcast.new
-    @podcast_links = [["iTunes", @podcast.itunes_url, cloudinary(IMAGE_LINK["iTunes".downcase.to_sym], 40, 90, "png")],
-                      ["Overcast", @podcast.overcast_url, cloudinary(IMAGE_LINK[name.downcase.to_sym], 40, 90, "png")],
-                      ["Android", @podcast.android_url, cloudinary(IMAGE_LINK[name.downcase.to_sym], 40, 90, "png")],
-                      ["RSS", @podcast.feed_url, cloudinary(IMAGE_LINK[name.downcase.to_sym], 40, 90, "png")]]
```

Comment:
No longer utilized, removed in https://github.com/forem/forem/commit/c0a7c17054c027b461280ff9fd641a5e5594ca6a#diff-55e44fa1f74c931894253048f567b8f5